src/screens/Login/styles.module.scss

```diff
@@ -0,0 +1,46 @@
+@import 'variables/colors';
+@import 'variables/animations';
+
+.login-wrapper {
+  background: $wild-sand;
+  border-top: 8px solid $cerulean;
+  margin: 52px auto;
+  max-width: 300px;
+  padding: 20px 24px 24px;
+  position: relative;
+  width: 100%;
+}
+
+.img {
+  display: block;
+  margin: 0 auto 16px;
+}
+
+.form {
+  gap: 16px;
+}
+
+.divider {
+  border-top: 2px solid $silver-chalice;
+}
+
+.overlay-container {
+  background-color: rgba($wild-sand, 0.8);
+  position: absolute;
+  z-index: 1;
```

Comment:
Since you have been sorting scss props it would be a shame to have a class that it's unsorted 👀